tests/test_shell.py

```diff
@@ -215,7 +561,15 @@ def test_shell(self):
             'print(str(project), job, len(list(jobs))); exit()', shell=True)
         assert out.strip() == '>>> {} None {}'.format(project, len(project))
 
+        out = self.call(
+            'python -m signac shell -c print(str(project),len(list(jobs)))'
+            .split())
```

Comment:
This is a little bit of a hack, it looks like you're avoiding spaces in the `-c` argument to prevent splitting that token. I suggest either specifying this as input or using another way to tokenize manually like:
```suggestion
        cmd = 'python -m signac shell -c'.split() + ['print(str(project), len(list(jobs)))']
        out = self.call(cmd)
```